moban/plugins/template.py

```diff
@@ -180,7 +183,8 @@ def expand_template_directories(dirs):
 def expand_template_directory(directory):
     log.debug("Expanding %s..." % directory)
     translated_directory = None
-    if ":" in directory and directory[1] != ":":
+    if ":" in directory and _is_windows_drive_notation(directory) is False:
```

Comment:
This is a bug here -- one letter repository names like https://github.com/jayvdb/z will break moban. https://github.com/moremoban/moban/issues/299

moban must either

- first check if the syntax is *valid* as a library name or as a remote-repo name, *before* looking for a local resource.

Or

- first check if the resource is a local path (abs or relative) *and* check if that local name is shadowing an online resource, *and* tell the user that the online resource is being shadowed, otherwise they are going to be surprised.

Either approach is terribly inefficient!  But it is consistent.  And moban can tell the user to add `../` or `./` prefixes which are invalid library names and invalid remote-repo name, so that allows moban to quickly choose a local resource.

Reply:
single character repo is addressed.

the syntax is:

1. repo_or_pypi_pkg_name:folder/subfolder
2. /unix/file/path/which/does/not/have/colon
3. C:/windows/path or C:\windows\path

first one is moban's own invention. last two are os dependent.